src/libs/actions/Session/index.ts

```diff
@@ -765,7 +777,7 @@ function invalidateCredentials() {
 
 function invalidateAuthToken() {
     NetworkStore.setAuthToken('pizza');
-    Onyx.merge(ONYXKEYS.SESSION, {authToken: 'pizza'});
+    Onyx.merge(ONYXKEYS.SESSION, {authToken: 'pizza', encryptedAuthToken: 'pizza'});
```

Comment:
Let's include the creationTime as well

Reply:
yes i pushed the version now

Reply:
there is a 50s lapse of time before the session is expired when clicked on the test tool. That will be enough for every tester to do te tests on native platforms comfortably.  I think you should redo your tests vids with the current version.

Reply:
Oh wait, why don't you add `creationTime` here? If so, if we use this tool, the session won't invalid.

Reply:
as i mentionned here https://github.com/Expensify/App/pull/53407#discussion_r1893843233 the point of this tool is to invalidate the session token in the BE and see how interactions go. No putting any a creationDate is the best choice to maintain the same experience with this tool before and after our changes, we are not impacting that tool